packages/swingset-liveslots/src/liveslots.js

```diff
@@ -1342,12 +1342,39 @@ function build(
     WeakSet: vom.VirtualObjectAwareWeakSet,
   });
 
+  function getRetentionStats() {
+    return {
+      ...collectionManager.getRetentionStats(),
+      ...vrm.getRetentionStats(),
+      ...vom.getRetentionStats(),
+      exportedRemotables: exportedRemotables.size,
+      importedDevices: importedDevices.size,
+      kernelRecognizableRemotables: kernelRecognizableRemotables.size,
+      exportedPromises: exportedVPIDs.size,
```

Comment:
These names are ok, but I'd be happy to have them be exactly the same as the object they describe (so `exportedVPIDs` instead of `exportedPromises`, `possiblyDeadSet`, `slotToVal`, etc). Those might be slightly shorter (and I find myself feeling that this record is going to take up a lot of space on the slogfile screen), and easier to map to the code during debugging. They're all already in a record called "stats", so we don't really need to make their names sound like counters.

Reply:
OK.  I've updated everything to use the collection name as the counter name universally.  It makes some of the counter names weird but does track with the implementation better.  I also updated the PR description above that documents the counters.